monitor/sample-monitor.conf

```diff
@@ -0,0 +1,14 @@
+[Teos]
+DATA_DIR = ~/.teos
+LOG_FILE = teos.log 
+CONF_FILE_NAME = teos.conf
+
+[Elastic]
+ES_HOST = localhost 
+ES_PORT = 9200
+KIBANA_HOST = localhost
+KIBANA_PORT = 9243
+
+# CLOUD_ID = System_monitor:sdiohafdlkjfl
```

Comment:
Can this last ones be deleted?

I guess there should be a kind of priority or check in the code.

If we go with priority, just prioritise one kind of connection over the other. If there's a check, check that only one is set, otherwise exit with an error.

PS: In case you decide to go with priority, at least warning the user so they are aware of both settings being active.